File: 01 CLASS PROF_LENIN/07_exe.py
print("Digite su nombre")
name = input().lower()
#vocales
vocales = "aeiou"
cantVocales = 0
for c in name:
    if c in vocales:
        cantVocales += 1
print(f"Su nombre tiene {cantVocales} vocales")
#invertido
name2 = name.upper()
nomInvertido = name2[::-1]
print(f"Su nombre invertido es : {nomInvertido}")
#en vertical
for c in name:
    print(c)
#en vertical invertido
for c in nomInvertido:
    print(c)

File: 01 CLASS PROF_LENIN/07_exe.cs
using System;

public class HelloWorld
{
    public static void Main(string[] args)
    {
        /*
        CASO 02
        Cantidad de vocales
        */
        Console.Write("Digite su nombre : ");
        string name = Console.ReadLine().ToLower();
        int vocalCant = 0;
        foreach (var c in name)
        {
        if ( c == 'a' || c == 'e' || c == 'i' || c == 'o' || c == 'u') {
            vocalCant++;
        }
        }
        Console.WriteLine($"Su nombre tiene {vocalCant} vocales");
        Console.ReadKey();
    }
}